Snakefile

```diff
@@ -285,6 +297,12 @@ rule download_pdbs:
         outdir = output_dir / foldseekclustering_dir
     benchmark:
         output_dir / benchmarks_dir / downloading_dir / "{acc}.download_pdbs.txt"
+    conda:
+        "envs/api.yml"
+    resources:
+        mem_mb=256
+    threads: 1
+    localrule: True
```

Comment:
This spawns hundreds of jobs that each last only a few seconds, so I set it as a `localrule` to avoid creating a new remote job for each task.

Reply:
clever! this is good to keep in mind as an option